src/frequenz/sdk/config/_config_managing.py

```diff
@@ -94,45 +139,72 @@ async def _run(self) -> None:
         """
         await self.send_config()
 
+        parent_paths = {p.parent for p in self._config_paths}
```

Comment:
`unique_parent_paths = [..]` for more clarity? :)

Reply:
I don't know, it is a set; is it really necessary to say that items in a set are unique?

Reply:
I just noticed your `[..]`, I wouldn't make it a list because it won't remove the duplicates and it won't be unique anymore. Even more, in that case it's just `non_unique_parent_paths_that_are_actually_the_same_as_config_paths =  list(self._config_paths)` :thinking: :laughing: